client/src/context/useBookingContext.tsx

```diff
@@ -0,0 +1,37 @@
+import { useContext, createContext, FunctionComponent, useState, useEffect } from 'react';
+import { Booking } from '../interface/Booking';
+import { fetchBookings, updateBooking } from '../helpers/APICalls/booking';
+
+interface IBookingContext {
```

Comment:
```suggestion
interface BookingContext {
```
We don't need to prefix interfaces with 'I' in JS